web/gds-admin-ui/src/components/ProfileDropdown.js

```diff
@@ -12,17 +13,22 @@ type ProfileDropdownState = {
     dropdownOpen?: boolean,
 };
 
+const api = new APICore()
+
 const ProfileDropdown = (props: ProfileDropdownProps, state: ProfileDropdownState): React$Element<any> => {
     const profilePic = props.profilePic || null;
     const [dropdownOpen, setDropdownOpen] = useState(false);
 
-    /*
-     * toggle profile-dropdown
-     */
     const toggleDropdown = () => {
         setDropdownOpen(!dropdownOpen);
     };
 
+    const handleLogoutClick = () => {
+        setTimeout(() => {
+            api.deleteUserSession()
+        }, 1000)
```

Comment:
Why the timeout of one second?

Reply:
Just to put a latency because I found the logout to fast

Reply:
What happens when it is too fast? It seems like we'd want it to be pretty immediate, particularly because this is security related.